### PR TITLE
Add coordinates

### PR DESCRIPTION
**Pull request by Sroose**
Coords returned when giving OPTION_INCLUDE_DEFAULT_GEO_LOCATION were not present in the return object

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you also update the tests?

File: src/AddressValidator.php [hunk 1]
```diff
@@ -46,6 +46,11 @@ public function validate(Address $address): ValidatedAddress
         $validateAddressesResponse = $this->gateway->validateAddresses(
             new ValidateAddressesRequest([$address], $this->options)
         );
+        
+         if(isset($validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError'])) {
+            $msg = 'BPOST ' . $validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError']['ErrorSeverity'] . ' ERROR: ' . $validateAddressesResponse->responseBody()['ValidateAddressesResponse']['GeneralError']['ErrorCode'];
```
Comment (sebastiandedeyne) [inline src/AddressValidator.php]: Please don't abbreviate variable names

File: src/Address.php [hunk 2]
```diff
@@ -21,6 +21,9 @@ class Address
 
     /** @var string */
     public $country = 'BELGIE';
+    
+    /** @var array */
+    public $coords;
```
Comment (sebastiandedeyne) [inline src/Address.php]: Please don't abbreviate variable names

File: src/AddressValidator.php [hunk 3]
```diff
@@ -46,6 +46,11 @@ public function validate(Address $address): ValidatedAddress
...
+            throw new \Exception($msg);
```
Comment (sebastiandedeyne) [inline src/AddressValidator.php]: Could you create a specific exception class for this? Similar to this one: https://github.com/spatie/bpost-address-webservice/blob/6066ee058aafd3fe17da782c9c3cf9aa05d7d711/src/AddressValidator.php#L58

**Pull request closed without merging**